mrcnn/config.py

```diff
@@ -210,9 +213,11 @@ def __init__(self):
 
         # Input image size
         if self.IMAGE_RESIZE_MODE == "crop":
-            self.IMAGE_SHAPE = np.array([self.IMAGE_MIN_DIM, self.IMAGE_MIN_DIM, 3])
+            self.IMAGE_SHAPE = np.array([self.IMAGE_MIN_DIM, self.IMAGE_MIN_DIM,
+                self.__class__.IMAGE_CHANNEL_COUNT])
         else:
-            self.IMAGE_SHAPE = np.array([self.IMAGE_MAX_DIM, self.IMAGE_MAX_DIM, 3])
+            self.IMAGE_SHAPE = np.array([self.IMAGE_MAX_DIM, self.IMAGE_MAX_DIM,
+                self.__class__.IMAGE_CHANNEL_COUNT])
```

Comment:
Is there a particular advantage to using self.__class__.IMAGE_CHANNEL_COUNT vs self.IMAGE_CHANNEL_COUNT?